### PR TITLE
Add lint check

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: database/database.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func (db *Database) GetModuleState(id string) (state []byte, err error) {
 	}
 	defer rows.Close()
 
-	for rows.Next() {
+	if rows.Next() {
```
Comment (al1img) [inline database/database.go:123]: Better would be:
```go
if !rows.Next() {
    return nil, aoserrors.New(ErrNotExistStr)
}
...
```

File: database/database.go [hunk 2]
```diff
@@ -160,7 +160,7 @@ func (db *Database) GetAosVersion(id string) (version uint64, err error) {
 	}
 	defer rows.Close()
 
-	for rows.Next() {
```
Comment (al1img) [inline database/database.go:163]: ditto

File: umclient/umclient.go [hunk 3]
```diff
@@ -165,7 +165,7 @@ func (client *Client) Close() (err error) {
 	log.Debug("Close UM client")
 
 	if client.stream != nil {
-		client.stream.CloseSend()
+		err = client.stream.CloseSend()
```
Comment (al1img) [inline umclient/umclient.go:168]: aoserror.Wrap

File: updatehandler/updatehandler.go [hunk 4]
```diff
@@ -340,7 +340,9 @@ func (handler *Handler) init() {
 		})
 	}
 
-	doPriorityOperations(operations, false)
+	if err := doPriorityOperations(operations, false); err != nil {
```
Comment (al1img) [inline updatehandler/updatehandler.go:343]: By design, this function never returns err in this case. Just void it:
```go
_ = doPriorityOperations
```
or change message to `Error init modules: %s`

File: updatehandler/updatehandler_test.go [hunk 5]
```diff
@@ -525,7 +528,9 @@ func TestUpdateSameAosVersion(t *testing.T) {
 	}
 
 	for _, element := range currentStatus.Components {
-		storage.SetAosVersion(element.ID, element.AosVersion)
+		if err := storage.SetAosVersion(element.ID, element.AosVersion); err != nil {
+			t.Errorf("Can't set aos version: %s", err)
```
Comment (al1img) [inline updatehandler/updatehandler_test.go:532]: aos -> Aos

File: updatehandler/updatehandler_test.go [hunk 6]
```diff
@@ -590,7 +595,9 @@ func TestUpdateWrongVersion(t *testing.T) {
...
```
Comment (al1img) [inline updatehandler/updatehandler_test.go:599]: ditto

File: updatemodules/boardconfigmodule/boardconfigmodule_test.go [hunk 7]
```diff
@@ -294,7 +294,7 @@ func createBoardImage(imagePath string, version string) (payload string, err err
 
 	payload = fmt.Sprintf(configTemplate, version)
 
-	zw.Write([]byte(payload))
+	_, err = zw.Write([]byte(payload))
```
Comment (al1img) [inline updatemodules/boardconfigmodule/boardconfigmodule_test.go:297]:
```go
if _, err = ...; err != nil {
    return nil, aoserros.Wrap(err)
}
```

File: updatemodules/partitions/controllers/ubootcontroller/ubootcontroller.go [hunk 8]
```diff
@@ -228,9 +221,7 @@ func (controller *UbootController) saveEnv() (err error) {
 
 	imagePath := path.Join(envMountPoint, controller.envFileName)
 
-	controller.cfg.SaveTo(imagePath)
-
-	return nil
+	return controller.cfg.SaveTo(imagePath)
```
Comment (al1img) [inline updatemodules/partitions/controllers/ubootcontroller/ubootcontroller.go:224]:
```go
if err = ...; err != nil {
    return aoserrors.Wrap(err)
}
```

File: updatemodules/partitions/modules/dualpartmodule/dualpartmodule_test.go [hunk 9]
```diff
@@ -74,12 +74,11 @@ type testChecker struct {
  ******************************************************************************/
 
 var tmpDir string
-var testImage string
 
 var disk *testtools.TestDisk
 
 var stateController = testStateController{}
-var stateStorage = testStateStorage{state: []byte("")}
+var stateStorage = testStateStorage{state: []byte("{}")}
```
Comment (al1img) [inline updatemodules/partitions/modules/dualpartmodule/dualpartmodule_test.go:81]: What kind of warning is in this line?

File: updatemodules/partitions/utils/efi/efi.go [hunk 10]
```diff
@@ -63,18 +63,13 @@ import (
...
-)
-
 const efiBootAbbrevHD = 2
 const eddDefaultDevice = 0x80
 
 const (
-	hdSignatureNone = iota
-	hdSignatureMBR
-	hdSignatureGUID
+	//hdSignatureNone = iota
```
Comment (al1img) [inline updatemodules/partitions/utils/efi/efi.go:70]: I don't like this fix. If there is no way to suppress this warning:
```go
        // hdSignatureNone = 0
	// hdSignatureMBR = 1
	hdSignatureGUID = 2
```

File: database/database.go (hunk 1 above)
Comment (Eabramov84) [reply database/database.go:123]: changed

File: database/database.go (hunk 2 above)
Comment (Eabramov84) [reply database/database.go:163]: changed

File: umclient/umclient.go (hunk 3 above)
Comment (Eabramov84) [reply umclient/umclient.go:168]: added

File: updatehandler/updatehandler.go (hunk 4 above)
Comment (Eabramov84) [reply updatehandler/updatehandler.go:343]: changed to _ = doPriorityOperations

File: updatehandler/updatehandler_test.go (hunk 5 above)
Comment (Eabramov84) [reply updatehandler/updatehandler_test.go:532]: done

File: updatehandler/updatehandler_test.go (hunk 6 above)
Comment (Eabramov84) [reply updatehandler/updatehandler_test.go:599]: done

File: updatemodules/boardconfigmodule/boardconfigmodule_test.go (hunk 7 above)
Comment (Eabramov84) [reply updatemodules/boardconfigmodule/boardconfigmodule_test.go:297]: done

File: updatemodules/partitions/controllers/ubootcontroller/ubootcontroller.go (hunk 8 above)
Comment (Eabramov84) [reply updatemodules/partitions/controllers/ubootcontroller/ubootcontroller.go:224]: done

File: updatemodules/partitions/modules/dualpartmodule/dualpartmodule_test.go (hunk 9 above)
Comment (Eabramov84) [reply updatemodules/partitions/modules/dualpartmodule/dualpartmodule_test.go:81]: we have missed check error in getState. The state should be in json format. Empty string returns json parsing error.

File: updatemodules/partitions/utils/efi/efi.go (hunk 10 above)
Comment (Eabramov84) [reply updatemodules/partitions/utils/efi/efi.go:70]: added warning suppresen  (//nolint)

File: updatemodules/partitions/utils/efi/efi.go [hunk 11]
```diff
@@ -63,18 +63,13 @@ import (
...
 const efiBootAbbrevHD = 2
 const eddDefaultDevice = 0x80
 
 const (
-	hdSignatureNone = iota
-	hdSignatureMBR
-	hdSignatureGUID
+	hdSignatureNone = iota //nolint
+	hdSignatureMBR         //nolint
+	hdSignatureGUID = 2
```
Comment (al1img) [inline updatemodules/partitions/utils/efi/efi.go:72]: = 2 is redundant

File: updatemodules/partitions/utils/efi/efi.go [hunk 12]
```diff
@@ -63,18 +63,13 @@ import (
...
-)
-
 const efiBootAbbrevHD = 2
 const eddDefaultDevice = 0x80
 
 const (
-	hdSignatureNone = iota
-	hdSignatureMBR
-	hdSignatureGUID
+	hdSignatureNone = iota //nolint
```
Comment (al1img) [inline updatemodules/partitions/utils/efi/efi.go:70]: can space be added after //

Comment (Eabramov84) [reply updatemodules/partitions/utils/efi/efi.go:70]: space added

**Review by al1img — APPROVED** [review]

**Pull request merged**